homeassistant/components/dynalite/translations/en.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This file is added automatically. No need to add it here

Reply:
ok. removed